architecture/topics/kuryr.adoc

```diff
@@ -37,12 +37,25 @@ access to the underlying OpenStack service APIs.
 [[architecture-additional-concepts-kuryr-cni]]
 === kuryr-cni
 
-kuryr-cni is (currently) a dummy container that is responsible for installing
-and configuring Kuryr CNI plug-in on {product-title} nodes. It should be
-running on every node, so it is modelled in {product-title} as `DaemonSet`.
-
-Kuryr CNI plug-in is called every time a new container that needs networking is
-spawned on the host. When that happens, the CNI plug-in starts watching
-{product-title} API to wait for kuryr-controller providing VIF annotation. When
-annotation is noticed, CNI plug-in uses this info to plug the VIF to the
-containers.
+kuryr-cni container serves two roles in Kuryr-Kubernetes deployment. It is
```

Comment:
{product-title} ?

Reply:
{product-title} inserts either OpenShift Origin or OpenShift Enterprise. In case of "Kuryr-Kubernetes" phrase we want to mention it explicitly as it's a product completely independent from OpenShift.